[2025-02-02T15:12:41.482+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-02T15:12:41.985+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_workers.test_task_2 manual__2025-02-02T15:11:47.551276+00:00 [queued]>
[2025-02-02T15:12:42.051+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_workers.test_task_2 manual__2025-02-02T15:11:47.551276+00:00 [queued]>
[2025-02-02T15:12:42.060+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-02-02T15:12:42.254+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): test_task_2> on 2025-02-02 15:11:47.551276+00:00
[2025-02-02T15:12:42.312+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=356) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-02T15:12:42.323+0000] {standard_task_runner.py:64} INFO - Started process 366 to run task
[2025-02-02T15:12:42.353+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'test_workers', 'test_task_2', 'manual__2025-02-02T15:11:47.551276+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmplfpp6ghh']
[2025-02-02T15:12:42.367+0000] {standard_task_runner.py:91} INFO - Job 5: Subtask test_task_2
[2025-02-02T15:12:42.782+0000] {task_command.py:426} INFO - Running <TaskInstance: test_workers.test_task_2 manual__2025-02-02T15:11:47.551276+00:00 [running]> on host aa23607e8c49
[2025-02-02T15:12:43.666+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='TUANTUAN' AIRFLOW_CTX_DAG_ID='test_workers' AIRFLOW_CTX_TASK_ID='test_task_2' AIRFLOW_CTX_EXECUTION_DATE='2025-02-02T15:11:47.551276+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-02T15:11:47.551276+00:00'
[2025-02-02T15:12:43.675+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-02T15:12:43.681+0000] {logging_mixin.py:188} INFO - ðŸ”¥ Task Task 2 Ä‘ang cháº¡y trÃªn worker: aa23607e8c49
[2025-02-02T15:12:46.685+0000] {logging_mixin.py:188} INFO - âœ… Task Task 2 Ä‘Ã£ hoÃ n thÃ nh trÃªn aa23607e8c49
[2025-02-02T15:12:46.688+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-02-02T15:12:46.715+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-02T15:12:46.830+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=test_workers, task_id=test_task_2, run_id=manual__2025-02-02T15:11:47.551276+00:00, execution_date=20250202T151147, start_date=20250202T151241, end_date=20250202T151246
[2025-02-02T15:12:47.002+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-02-02T15:12:47.246+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-02T15:12:47.259+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
